generic/samples/native-client/src/nativeMain/kotlin/app.kt

```diff
@@ -0,0 +1,11 @@
+import io.ktor.client.engine.curl.Curl
+import io.ktor.util.*
+import kotlinx.coroutines.runBlocking
+
+@KtorExperimentalAPI
+fun main() = runBlocking {
+    // To prevent non-zero exit status because of detected memory leaks
+    Platform.isMemoryLeakCheckerActive = false
```

Comment:
Not entirely clear if this is something that's always needed or to avoid a bug with Ktor or...

Reply:
Without it, I get `Unfinished workers detected, 1 workers leaked!` message and 134 exit code. Should I file an issue?

Reply:
I'd say yes please. Because it's not normal for someone to know they have to set a memory leak checker. We either need it documented or resolved in code. And same as the above, I'd link to the corresponding issue.